File: client/iih_stub/iih.ci_stub/iih.ci.ord/d/CijudgeSpecillDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.d
{
    /// <summary>
    /// CijudgeSpecillDTO 的摘要说明。
    /// </summary>
    public class CijudgeSpecillDTO : BaseDTO {

        public CijudgeSpecillDTO() {
 
        }

        /// <summary>
        /// 药品
        /// </summary>
		public string Id_mm {
            get { return getAttrVal<string>("Id_mm",null); }
            set { setAttrVal<string>("Id_mm", value); }
        }

        /// <summary>
        /// 特殊病药
        /// </summary>
		public bool? Fg_mmspecill {
            get { return getAttrVal<FBoolean>("Fg_mmspecill",null); }
            set { setAttrVal<FBoolean>("Fg_mmspecill", value); }
        }

        /// <summary>
        /// 诊断集合
        /// </summary>
		public FMap2 Diagmap {
            get { return getAttrVal<FMap2>("Diagmap",null); }
            set { setAttrVal<FMap2>("Diagmap", value); }
        }

        /// <summary>
        /// 扩展字段
        /// </summary>
		public string Str {
            get { return getAttrVal<string>("Str",null); }
            set { setAttrVal<string>("Str", value); }
        }

        /// <summary>
        /// 扩展字段2
        /// </summary>
		public string Str2 {
            get { return getAttrVal<string>("Str2",null); }
            set { setAttrVal<string>("Str2", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_mm", "Fg_mmspecill", "Diagmap", "Str", "Str2"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.d.CijudgeSpecillDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/d/CijudgeSpecillDTO.java
package iih.ci.ord.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * ci判断特殊病药 DTO数据 
 * 
 */
public class CijudgeSpecillDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 药品
	 * @return String
	 */
	public String getId_mm() {
		return ((String) getAttrVal("Id_mm"));
	}
	/**
	 * 药品
	 * @param Id_mm
	 */
	public void setId_mm(String Id_mm) {
		setAttrVal("Id_mm", Id_mm);
	}
	/**
	 * 特殊病药
	 * @return FBoolean
	 */
	public FBoolean getFg_mmspecill() {
		return ((FBoolean) getAttrVal("Fg_mmspecill"));
	}
	/**
	 * 特殊病药
	 * @param Fg_mmspecill
	 */
	public void setFg_mmspecill(FBoolean Fg_mmspecill) {
		setAttrVal("Fg_mmspecill", Fg_mmspecill);
	}
	/**
	 * 诊断集合
	 * @return FMap2
	 */
	public FMap2 getDiagmap() {
		return ((FMap2) getAttrVal("Diagmap"));
	}
	/**
	 * 诊断集合
	 * @param Diagmap
	 */
	public void setDiagmap(FMap2 Diagmap) {
		setAttrVal("Diagmap", Diagmap);
	}
	/**
	 * 扩展字段
	 * @return String
	 */
	public String getStr() {
		return ((String) getAttrVal("Str"));
	}
	/**
	 * 扩展字段
	 * @param Str
	 */
	public void setStr(String Str) {
		setAttrVal("Str", Str);
	}
	/**
	 * 扩展字段2
	 * @return String
	 */
	public String getStr2() {
		return ((String) getAttrVal("Str2"));
	}
	/**
	 * 扩展字段2
	 * @param Str2
	 */
	public void setStr2(String Str2) {
		setAttrVal("Str2", Str2);
	}
}